src/UKHO.S100PermitService.Common/Services/ManufacturerKeyService.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Azure.Security.KeyVault.Secrets;
+using Microsoft.Extensions.Options;
+using UKHO.S100PermitService.Common.Clients;
+using UKHO.S100PermitService.Common.Configuration;
+using UKHO.S100PermitService.Common.Events;
+using UKHO.S100PermitService.Common.Exceptions;
+using UKHO.S100PermitService.Common.Providers;
+
+namespace UKHO.S100PermitService.Common.Services
+{
+    public class ManufacturerKeyService : IManufacturerKeyService
+    {
+        private readonly IOptions<ManufacturerKeyConfiguration> _manufacturerKeyvault;
```

Comment:
Please rename to _manufacturerKeyVault from _manufacturerKeyvault

Reply:
Done